docs/pages/monitoring/nuts-consent-bridge.rst

```diff
@@ -0,0 +1,18 @@
+.. _nuts-consent-bridge-monitoring:
+
+Nuts consent bridge monitoring
+##############################
+
+Basic service health
+********************
+
+A status endpoint is provided to check if the service is running and if the web server has been started.
+The endpoint is available over http so it can be used by a wide range of health checking services.
+It does not provide any information on the individual engines running as part of the executable.
+The main goal of the service is to give a YES/NO answer for if the service is running?
+
+.. code-block::
+
+    GET /status
+
+It'll return an empty response with a 200 status code.
```

Comment:
Although no-one will (probably) care, shouldn't it be 201 (no content) when the response is empty?

Reply:
Should be yes, But some status checkers are quite picky and use 200 as default...

Reply:
Hmmm better to add the "OK" string in the body then everything is up to standards again

Reply:
Better still, conform to the Spring Boot health indicator response:
```
{
    "status" : "UP"
}
```